server/meshmodel/relationships/binding.json

```diff
@@ -0,0 +1,43 @@
+{
+    "apiVersion": "core.meshery.io/v1alpha1",
+    "kind": "Binding",
+    "metadata": {
+      "description": "A relationship between two resources where one resource uses the another resource for it's basic or enhanced functionality or Integrity.",
+      "name": "Binding Relationship",
+      "type": "Binding"
+    },
+    "model": {
+      "name": "kubernetes",
+      "version": "v1.25.2",
```

Comment:
@MUzairS15 are we able to make this version optional? Or is it essentially inconsequential at this juncture?